public/app/percona/inventory/Tabs/Agents.tsx

```diff
@@ -51,10 +52,22 @@ export const Agents: FC<GrafanaRouteComponentProps<{ serviceId: string; nodeId:
   const { isLoading: nodesLoading, nodes } = useSelector(getNodes);
   const styles = useStyles2(getStyles);
 
+  const mappedNodes: Node[] = useMemo(
+    (): Node[] => {
+      if(nodes.length > 0) {
```

Comment:
I think this check is a bit redundant, I think you can do just:

```tsx
const mappedNodes = useMemo(() => nodeFromDbMapper(nodes)...etc, [nodes);
```

For an empty array the body of the map function in `nodeFromDbMapper` will not run.